### ISSUE TITLE
Add cy.each method to the API

### ISSUE THREAD

**Issue by bahmutov** [cypress-io/cypress#156]
We wanted to validate / perform actions for a each element in the list, and for this we used the suggested manual code
```js
cy.get("li").then(function($lis){
  $lis.each(function(i, el){
    cy.wrap(Cypress.$(el)).click()
  })
})
```
Seems, this has a little boilerplate that can be avoided if there was a built-in `cy.each` method. It would work on a list of objects returned by `cy.get()` or `cy.find()` and would automatically wrap each element
```js
cy.get("li")
  .each(function($el){
    // $el is wrapped element
    $el.click()
  })
```

**Comment by brian-mann** [comment]
We can add this however `$el` would not be a Cypress wrapped element, it would just be a jQuery wrapped element - which would make it easier to `cy.wrap`.

So this removes the boilerplate of manually `.each`ing each element and then needing to instantiate jquery instances. 

```javascript
cy.get("li")
  .each(function($el){
    // $el is wrapped jquery element
    cy.wrap($el).click()
  })
```

**Comment by bahmutov** [comment]
sure, it could be just jQuery wrapped element, not `cy.wrap`ped. In our case we do additional steps per element, so clicking is just example operator, agreed

**Comment by brian-mann** [comment]
This is done and will be released with `0.16.2`

**Comment by brian-mann** [comment]
Fixed in [`0.16.2`](https://github.com/cypress-io/cypress/wiki/Changelog#0162-06112016).
